taxi_service/urls.py

```diff
@@ -22,4 +22,5 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("accounts/", include("django.contrib.auth.urls"))
```

Comment:
it was about whole accounts app
you have login/logout urls from django.contrib.auth.urls, haven't you?